Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+@objc(TRDailyForecast) class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+
+    init?(JSON: AnyObject?) {
```

Comment:
Probably not applicable to this PR, but do you know the reason we're not using Argo?